### ISSUE TITLE
Could not resolve import "cheerio/lib/slim"

### ISSUE THREAD

**Issue by igorgusarov** [Baroshem/nuxt-security#514]
### Version

nuxt-security: 1.4.3
nuxt: 3.13.0

### Issue

After updating Nuxt to 3.13.0, nuxt-security triggers a bunch of errors:
```[plugin node-resolve] Could not resolve import "cheerio/lib/slim" in /my/app/path/node_modules/.pnpm/nuxt-security@1.4.3_magicast@0.3.4_rollup@4.21.0/node_modules/nuxt-security/dist/runtime/nitro/plugins/40-preprocessHtml.mjs using exports defined in /Users/igor/Sites/platform/apps/console/web/node_modules/.pnpm/cheerio@1.0.0/node_modules/cheerio/package.json.```

Cheerio recently got updated to 1.0.0, and this update includes changing paths from `cheerio/lib/slim` to `cheerio/slim`: https://github.com/cheeriojs/cheerio/releases/tag/v1.0.0

**Comment by Amar-Gill** [comment]
In the mean time I am using a package override in my `package.json` file to get around this.

I've added the following:
```json
// package.json
  "pnpm": {
    "overrides": {
      "nuxt-security>cheerio": "1.0.0-rc.12"
    }
  }
```

**Comment by miclgael** [comment]
Thanks so much @Amar-Gill for providing the work around. 🙏

**Comment by Harm-Nullix** [comment]
How dumb that v1.0.0 crashes xD, just after the canidate is accepted..!?

Oh well.. been there..

package.json workaround:
```json
  "resolutions": {
    "cheerio": "1.0.0-rc.12"
  }
```

*If this is not working, try `rm -rf node_modules && rm -f yarn.lock` or something*

**Comment by Harm-Nullix** [comment]
Awesome!

Hope you had a good holiday!
Good job of not looking at your mail / git 💯

**Comment by Baroshem** [comment]
Released in 2.0.0 :)
